beets/autotag/mb.py

```diff
@@ -28,9 +28,14 @@
 from beets import util
 from beets import config
 import six
+import sys
 
 VARIOUS_ARTISTS_ID = '89ad4ac3-39f7-470e-963a-56509c546377'
-BASE_URL = 'http://musicbrainz.org/'
+
+if sys.version_info > (2, 7, 9):
```

Comment:
The comparison should "greater than or equal to", since 2.7.9 also has support for it.